src/classes/example.py

```
@@ -0,0 +1,136 @@
+"""
+Example script for demonstrating the web scraping task distribution system
+"""
+import time
+import random
+from datetime import datetime
+
+from process import Process
+from models.enums import TaskStatus, WorkerStatus, DistributionStrategy
+from models.task import Task
```

Ruff (0.8.2)

`models.task.Task` imported but unused

Remove unused import: `models.task.Task`

(F401)